pkg/util/util.go

```diff
@@ -49,8 +47,34 @@ var (
 	isAlphanumericRegex = regexp.MustCompile(`^[a-zA-Z0-9]*$`).MatchString
 	// MAC Address Regex Source: https://stackoverflow.com/a/4260512
 	isMACAddressRegex = regexp.MustCompile(`([0-9A-Fa-f]{2}[:-]){5}([0-9A-Fa-f]{2})`)
+
+	// DriverName is the domain for all EBS CSI Driver related components.
+	// Variable instead of constant to allow initialization from plugin.
+	driverName = ""
 )
 
+func SetDriverName(name string) {
+	if driverName != "" {
+		klog.ErrorS(nil, "Attempted to set driver name multiple times")
+		klog.FlushAndExit(klog.ExitFlushTimeout, 0)
+	}
```

Comment:
I really like the idea of flushing and exiting here; I suggest instead of checking `if driverName != ""` we use a synchronization primitive (sync.Once) to guarantee this is executed exactly once.